edd_utils/__init__.py

```diff
@@ -137,6 +137,71 @@ def export_study(session, slug, edd_server='edd.jbei.org', verbose=True):
     return df
 
 
+#Get metadata
+def export_metadata(session, slug, edd_server='edd.jbei.org', verbose=True):
+    '''Export a Study from EDD as a pandas dataframe'''
+
+    try:
+        lookup_response = session.get(f'https://{edd_server}/rest/studies/?slug={slug}')
+
+    except KeyError:
+        if lookup_response.status_code == requests.codes.forbidden:
+            print('Access to EDD not granted\n.')
+            sys.exit()
+        elif lookup_response.status_code == requests.codes.not_found:
+            print('EDD study was not found\n.')
+            sys.exit()
+        elif lookup_response.status_code == requests.codes.server_error:
+            print('Server error\n.')
+            sys.exit()
+        else:
+            print('An error with EDD export has occurred\n.')
+            sys.exit()
+
+    json_response = lookup_response.json()
+    # Catch the error if study slug is not found in edd_server
+    try: 
+        study_id = json_response["results"][0]["pk"]
+    except IndexError:
+        if json_response["results"] == []:
+            print(f'Slug \'{slug}\' not found in {edd_server}.\n')
+            sys.exit()
+    # TODO: catch the error if the study is found but cannot be accessed by this user
+    
+    # Get the metadata value's
+    export_response = session.get(f'https://{edd_server}/rest/lines/?study={study_id}')
+    metadata=export_response.json()
+    metadata=metadata['results'][-1]["metadata"]
```

Comment:
This is only getting the metadata for the last line returned. Different lines can have different sets of metadata, should merge the keys to get a full list on the study.